src/main/java/com/google/api/codegen/transformer/ApiCallableTransformer.java

```diff
@@ -79,7 +80,11 @@ public ApiCallableTransformer() {
     apiCallableBuilder.memberName(context.getNamer().getSettingsMemberName(method));
     apiCallableBuilder.settingsFunctionName(context.getNamer().getSettingsFunctionName(method));
 
-    if (methodConfig.isBundling()) {
+    if (methodConfig.isGrpcStreaming()) {
+      apiCallableBuilder.type(ApiCallableType.StreamingApiCallable);
+      apiCallableBuilder.streamingType(methodConfig.getGrpcStreaming().getType());
+      typeTable.saveNicknameFor("io.grpc.stub.StreamObserver");
```

Comment:
This is Java-specific, so it should either be made more generic (go through SurfaceNamer) or should be moved somewhere else. (Likewise for the same import further down)